ddp-workspace/projects/ddp-sdk/src/lib/components/address/addressEmbedded.component.ts

```diff
@@ -225,59 +234,74 @@ export class AddressEmbeddedComponent implements OnDestroy, OnInit {
     const busyCounter$ = new BehaviorSubject(0);
 
     const isThisComponentBusy$ = busyCounter$.pipe(
-      scan((acc, val) => acc + val, 0),
-      map(val => val > 0),
-      distinctUntilChanged()
+        scan((acc, val) => acc + val, 0),
+        map(val => val > 0),
+        distinctUntilChanged()
     );
 
     const initializeStateAction$ = this.state$.pipe(
-      take(1),
-      tap(() => busyCounter$.next(1)),
-      mergeMap((state) => this.addressService.findDefaultAddress().pipe(
-        map(defaultAddress => [state, defaultAddress]))
-      ),
-      tap(([state, defaultAddress]) =>
-        defaultAddress && this.stateUpdates$.next({ inputAddress: defaultAddress as Address })),
-      // filter for case where we need to go on to look for a temp address?
-      filter(([state, defaultAddress]) => !defaultAddress && !!(state as ComponentState).activityInstanceGuid),
-      map(([state, _]) => state as ComponentState),
-      mergeMap((state) => this.addressService.getTempAddress(state.activityInstanceGuid)),
-      tap((tempAddress) => tempAddress && this.stateUpdates$.next({ inputAddress: tempAddress as Address })),
-      // fake that the address was just entered. Perhaps this can become a separate subject?
-      // guess we are saving temp address again. No harm but not nice either.
-      tap((tempAddress) => this.inputComponentAddress$.next(tempAddress)),
-      finalize(() => busyCounter$.next(-1))
+        take(1),
+        tap(() => busyCounter$.next(1)),
+        // let's disable form while we try to load initial address
+        tap(() => this.stateUpdates$.next({isTemporarilyDisabled: true})),
+        mergeMap((state) => this.addressService.findDefaultAddress().pipe(
+            map(defaultAddress => [state, defaultAddress]))
+        ),
+        tap(([_, defaultAddress]: [ComponentState, Address | null]) =>
+            defaultAddress && this.inputAddress$.next(defaultAddress)),
+        // filter for case where we need to go on to look for a temp address?
+        filter(([state, defaultAddress]) => !defaultAddress && !!(state as ComponentState).activityInstanceGuid),
+        // map(([state, _]) => state as ComponentState),
+        mergeMap(([state, _]) => this.addressService.getTempAddress(state.activityInstanceGuid)),
+        tap((tempAddress) => this.inputAddress$.next(tempAddress)),
+        // fake that the address was just entered. Perhaps this can become a separate subject?
+        // guess we are saving temp address again. No harm but not nice either.
+        tap((tempAddress) => this.inputComponentAddress$.next(tempAddress)),
+        finalize(() => {
+            this.stateUpdates$.next({isTemporarilyDisabled: false});
+            busyCounter$.next(-1);
+        }),
     );
```

Comment:
Does simlar stuff as before: check if we have a default address, if not, check if we have a temp address. Now we pass it on to the input component via the inputAddress$ subject. We are also disabling temporarily.